app/src/main/java/com/example/gravit/main/User/MyPage.kt

```
@@ -407,8 +543,18 @@ fun SummaryUI(
                 }
-                val rankInfo =
-                    listOf("${summaries?.learningSummary?.completedLessonCount}개", "완료 레슨", "${String.format("%.1f", summaries?.learningSummary?.totalLearningHours)}h", "총 학습시간", "${summaries?.learningSummary?.averageAccuracy}%", "평균 정답률").chunked(2)
+                val rankInfo = listOf(
+                    summaries?.learningSummary?.averageAccuracy?.let { "${it}%" } ?: "-",
+                    "평균 정답률",
+
+                    summaries?.learningSummary?.completedLessonCount?.let { "${it}개" } ?: "-",
+                    "완료 레슨",
+
+                    summaries?.learningSummary?.totalLearningHours?.let {
+                        String.format("%.1fh", it)
```

detekt (1.23.8)

[warning] String.format("%.1fh", it) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)